go.mod

```diff
@@ -2,4 +2,13 @@ module github.com/PaddleHQ/modfmt
 
 go 1.24.0
 
-require golang.org/x/mod v0.24.0
+require (
+	github.com/stretchr/testify v1.10.0
```

Comment:
I like testify - I know it's controversial - but it does make code much more readable